src/components/include/application_manager/application_manager.h

```diff
@@ -93,6 +93,7 @@ class StateControllerImpl;
 struct CommandParametersPermissions;
 using policy::RPCParams;
 typedef std::vector<ApplicationSharedPtr> AppSharedPtrs;
+enum class LockScreenDismissalEnabled { kNotExists = -1, kDisabled, kEnabled };
```

Comment:
@IGapchuk rename to LockScreenDismissalState.

Reply:
@LuxoftAKutsan  Fixed in cd078272812e56dbc954d3d9319ac822f24b3093.